### ISSUE TITLE
Add support for `variant`

### ISSUE THREAD

**Issue by nilsreichardt** [eBay/flutter_glove_box#179]
The `testWidgets` method allows specifying `variant` parameter. This is helpful when you have different design for different platforms.

**Comment by NickalasB** [comment]
HI @nilsreichardt. That doesn't seem like too crazy of a change. Would you be interested in opening up a PR to add the variant param to the `testGoldens` function?

**Comment by nilsreichardt** [comment]
There is already an open PR. But the author seems to be inactive. Should I create a new PR or write a comment under the PR?

**Comment by NickalasB** [comment]
Yeah I think making a new PR that is limited to just adding the `variant` param would be easiest honestly
